astro/src/content/docs/operate/deploy/proxy-setup.mdx

```diff
@@ -145,9 +145,10 @@ This guide covers proxying requests *to* FusionAuth, in order to add a layer of
 
 If you are self-hosting, you can also proxy requests *from* FusionAuth, such as [webhooks](/docs/extend/events-and-webhooks/) or [connector requests](/docs/lifecycle/migrate-users/connectors/). You can read more about that in the [Configuration Reference](/docs/reference/configuration). Look for the `proxy.*` configuration values. This functionality is not available in FusionAuth Cloud.
 
-## SNI and FusionAuth Cloud Instances
+## FusionAuth Cloud Instances
 
-The latest FusionAuth Cloud Instances will use [SNI](https://en.wikipedia.org/wiki/Server_Name_Indication) to negotiate a TLS connection. If you are using a proxy in front of your FusionAuth instance, ensure it supports the use of SNI for TLS connections.
+The latest FusionAuth Cloud Instances will use [SNI](https://en.wikipedia.org/wiki/Server_Name_Indication) to negotiate a TLS connection. If you are using a proxy in front of your FusionAuth instance, ensure it supports the use of SNI for TLS connections. Additionally, to allow for proper TLS certificate handling
+the `Host` header should be set to the domain name of your FusionAuth instance (e.g. `example.fusionauth.io`).
```

Comment:
Remove this line (moved to the line above; there was an extra newline in the middle of the sentence).